conf/application.conf

```diff
@@ -79,6 +79,13 @@ controllers {
       pattern = "/([\\w-]+)/([^/]+)[/]?.*"
     }
   }
+  controllers.ATSPAYEDataController = {
+    needsAuth = false
+    authParams = {
```

Comment:
Do you need auth params when auth is set to false?

Reply:
Don't know... First pass... Will change this later if needs be.

Reply:
If they are needed then should the account value be set to `paye` and  not `sa`